src/v4/screen/StoryTargetSelection/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import AmityStoryTargetSelectionPage from '../../PublicApi/Pages/AmityStoryTargetSelectionPage/AmityStoryTargetSelectionPage';
+
+const StoryTargetSelection = () => {
+  return <AmityStoryTargetSelectionPage />;
+};
+
+export default StoryTargetSelection;
```

Comment:
This file will be same as AmitySocialHomePage. 
If we can use as component like this <StoryTargetSelection /> , don't need to make a new screen. 
I will approve now.